yoyodyne/defaults.py

```diff
@@ -14,6 +14,7 @@
 # Architecture arguments.
 ARCH = "attentive_lstm"
 ATTENTION_HEADS = 4
```

Comment:
I think for clarity I'd prefer `SOURCE_ATTENTION_HEADS` and `FEATURES_ATTENTION_HEADS` (note for consistency I am always using "features" and not "feature"). And same for the flags. We do this with encoder modules too---distinguish between source and features even though sometimes source includes features because concatenation...

Reply:
It may be that `SOURCE_ATTENTION_HEADS` is the best name, but wanted to clarify that we reuse this variable for self attention in both the encoder and decoder, as well as for the "cross-attention" (can't remember the canonical name for this) from encoder to decoder. I think using the same number for all of these is standard. SOURCE could imply that it is specific to the encoder, or just the "cross" attention?

Reply:
I see and agree. Not a bad idea to comment that somewhere in the source (maybe not here though).